### DISCUSSION TITLE
importing External js file

### DISCUSSION THREAD

**Discussion by akhil-c-k** [codesandbox/codesandbox-client#4573] [General]
i didn't understand why  refference error occurs when i use external js , i mean if i create one js file in folder and if i import in index.html using script tag then refference error occuring for the method i defined.

`https://codesandbox.io/s/quirky-hill-6wtoq?file=/index.html`

**Comment by CodeSandboxTeam** [comment]
Hey! 


This should definitely work. Can you recreate the error in your sandbox again so that I can have a look?


Thanks!



--- original message ---
On July 18, 2020, 12:00 PM GMT+2 notifications@github.com wrote:



i didn't understand why refference error occurs when i use external js , i mean if i create one js file in folder and if i import in index.html using script tag then refference error occuring for the method i defined

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---
